tests/UKHO.S100PermitService.API.FunctionalTests/Helpers/TestConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace UKHO.S100PermitService.API.FunctionalTests.Helpers
+{
+    public class TestConfiguration
+    {
+        protected IConfigurationRoot _configurationRoot;
+        public PermitServiceApiConfiguration PermitServiceConfig { get; private set; }
```

Comment:
Where is this instantiated? Do you need to create a default instance in the ctor()?

Reply:
Sorry, missed pushing this change. Have push the required code in recent commit.